signup/signupController.js

```diff
@@ -158,6 +158,7 @@ define('signupController', [], function() {
 		};
 
 		$scope.residentialAppointment = function() {
+			$scope.questions[6] = null;
```

Comment:
I'm assuming this has to be accessed by its index?

Reply:
Yah, at least it's what the other ones do above. Theoretically, I think we could assign an ID to it in HTML and like query that somehow, but I think that is more work than it's worth for this code since the questions don't really ever change